tcms/telemetry/static/telemetry/js/testing/status-matrix.js

```diff
@@ -0,0 +1,117 @@
+$(document).ready(() => {
+
+    loadInitialProduct();
+
+    document.getElementById('id_product').onchange = () => {
+        update_version_select_from_product();
+        update_build_select_from_product();
+        updateTestPlanSelectFromProduct();
+
+        redrawTable();
+    };
+
+    document.getElementById('id_version').onchange = redrawTable;
+    document.getElementById('id_build').onchange = redrawTable;
+    document.getElementById('id_test_plan').onchange = redrawTable;
+
+    $('#id_after').on('dp.change', redrawTable);
+    $('#id_before').on('dp.change', redrawTable);
+
+    drawTable();
+});
+
+function drawTable(query = {}) {
+    jsonRPC('Testing.status_matrix', query, data => {
+        const testRunIds = [];
+
+        data.test_runs.forEach(testRun => {
+            const testRunHeader = d3.select('table > thead > tr')
+                .append('td');
+
+            testRunHeader.append('a')
+                .text(testRun.summary)
+                .attr('href', `${window.location.origin}/plan/${testRun.run_id}/`);
```

Comment:
url is wrong.

Reply:
what should it be? i'm testing it and it opens the execution tab in the run page